packages/styles/scss/components/logo-grid/_logo-grid.scss

```diff
@@ -139,7 +139,7 @@
   .#{$prefix}--logo-grid
     .#{$prefix}--content-block__cta
     .#{$prefix}--card__heading {
-    @include carbon--type-style('productive-heading-01');
+    @include carbon--type-style('expressive-heading-02');
```

Comment:
@BrunnoM7 if this is the React version of LogoGrid, it might be an issue with the way the card link is used. @shixiedesign identified that the card link's heading in general was incorrect, and I noticed that `CardLink` was passing in `copy` instead of `heading`. See PR here: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/7056.